src/js/views/OperationsLog/Display.jsx

```diff
@@ -1,12 +1,20 @@
 import PropTypes from 'prop-types'
-import React from 'react'
+import React, { useContext } from 'react'
 import { DateTime } from 'luxon'
 import { Icon, Markdown } from '../../components'
 import { useTranslation } from 'react-i18next'
 import { DescriptionList } from '../../components/DescriptionList/DescriptionList'
 import { Definition } from '../../components/DescriptionList/Definition'
+import { Context } from '../../state'
+
+function renderURLTemplate(urlTemplate, slug, environment) {
+  return urlTemplate
+    .replace('{environment}', environment.toString().toLowerCase())
+    .replace('{slug}', slug.toString().toLowerCase())
+}
```

Comment:
we might want to adopt something like https://www.npmjs.com/package/uri-templates in the future... it is similar to what I plan on using for server-side URL templating